internal/engine/upper_test.go

```diff
@@ -1548,6 +1549,11 @@ func (f *testFixture) WaitUntil(msg string, isDone func(store.EngineState) bool)
 
 		select {
 		case <-ctx.Done():
+			// dump the stacks of all goroutines
+			pprof.Lookup("goroutine").WriteTo(os.Stdout, 1)
```

Comment:
did you mean to leave this here?

Reply:
Yeah, I found it useful when debugging deadlocked tests. Do you think it's likely to be more noisy than useful?